tests/sysprep_test.go

```diff
@@ -251,7 +251,12 @@ var _ = Describe("[Serial][Sysprep][sig-compute]Syspreped VirtualMachineInstance
 		const OSWindowsSysprep = "windows-sysprep"
 		virtClient = kubevirt.Client()
 		libstorage.CreatePVC(OSWindowsSysprep, testsuite.GetTestNamespace(nil), "35Gi", libstorage.Config.StorageClassWindows, true)
-		answerFileWithKey := insertProductKeyToAnswerFileTemplate(answerFileTemplate)
+
+		// TODO: Figure out why we have both Autounattend and Unattend
+		cm := libconfigmap.New("sysprepautounattend", map[string]string{"Autounattend.xml": insertProductKeyToAnswerFileTemplate(answerFileTemplate), "Unattend.xml": insertProductKeyToAnswerFileTemplate(answerFileTemplate)})
+		_, err := virtClient.CoreV1().ConfigMaps(windowsVMI.Namespace).Create(context.Background(), cm, metav1.CreateOptions{})
```

Comment:
`windowsVMI.Namespace` is empty at this stage

Reply:
Thanks